.ci/Jenkinsfile

```diff
@@ -207,79 +109,6 @@ pipeline {
-    stage('Release') {
-      options { skipDefaultCheckout() }
-      environment {
-        HOME = '/var/lib/jenkins'
-      }
-      when {
-        beforeAgent true
-        tag pattern: 'v\\d+.*', comparator: 'REGEXP'
-      }
-      stages {
-        stage('Notify') {
-          steps {
-            notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Release *${env.TAG_NAME}* ready to be pushed",
-                         body: "(<${env.RUN_DISPLAY_URL}|Open>).\n Changes: ${env.TAG_NAME}")
-          }
-        }
-        stage('Release') {
-          steps {
-            deleteDir()
-            unstash 'source'
-            prepareRelease(){
-              sh 'rake release'
-            }
-          }
-        }
-        stage('Update Branch') {
-          steps {
-            deleteDir()
-            unstash 'source'
-            prepareRelease(){
-              catchError(buildResult: 'SUCCESS', message: 'Update branch task failed', stageResult: 'UNSTABLE') {
-                sh 'rake release:update_branch'
-              }
-            }
-          }
-          post {
-            success {
-              notifyStatus(slackStatus: 'good', subject: "[${env.REPO}] Release *${env.TAG_NAME}* published",
-                           body: "Great news, the release has been done successfully. (<${env.RUN_DISPLAY_URL}|Open>).")
-            }
-            unsuccessful {
-              notifyStatus(slackStatus: 'warning', subject: "[${env.REPO}] Release *${env.TAG_NAME}* has some sync-up post-release failures.",
-                           body: "Please go to ${env.BUILD_URL} to review the logs. Most likely you need to update the branch manually. (<${env.RUN_DISPLAY_URL}|Open>)")
-            }
-          }
-        }
-        stage('Opbeans') {
-          environment {
-            REPO_NAME = "${OPBEANS_REPO}"
-            VERSION = "${env.BRANCH_NAME.replaceAll('^v', '')}"
-          }
-          steps {
-            deleteDir()
-            // Let's wait for the Gem to be available
-            sh label: 'Wait for gem', script: """#!/usr/bin/env bash
-              source /usr/local/bin/bash_standard_lib.sh
-              (retry 10 curl --silent --show-error --fail -I https://rubygems.org/gems/elastic-apm/versions/${env.VERSION})
-            """
-            dir("${OPBEANS_REPO}"){
-              git(credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba',
-                  url: "git@github.com:elastic/${OPBEANS_REPO}.git",
-                  branch: 'main')
-              // It's required to transform the tag value to the gem version
-              sh script: ".ci/bump-version.sh ${env.VERSION}", label: 'Bump version'
-              // The opbeans pipeline will trigger a release for the main branch
-              gitPush()
-              // The opbeans pipeline will trigger a release for the release tag
-              gitCreateTag(tag: "${env.BRANCH_NAME}")
-            }
-          }
-        }
-      }
-    }
   }
   post {
```

Comment:
There is no need to run the post-cleanup even though if the `micro-benchmarks` have not been migrated yet.

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1366/commits/cffeaf6ad2fdd32365e9198883754b6e6bf2390e